frontend/src/AlertsManagement/AlertsContent.test.js

```diff
@@ -7,7 +7,7 @@ import { createShallow } from '@material-ui/core/test-utils';
 import { PureAlertsContent as AlertsContent } from "./AlertsContent";
 import AlertsList from "./AlertsList";
 
-configure({ adapter: new Adapter() });
+configure({ adapter: new Adapter(), disableLifecycleMethods: true });
```

Comment:
ooc what does `disableLifecycle...` do?

Reply:
with Enzyme when you (shallow) mount a component it will also run the `componentDidMount` function (and other lifecycle functions). but b/c I want to test the `handleTabs` and `handleCheckbox` functions with fake data I want to set the states myself instead of trying to run fetch and set the states (like it does in `componentDidMount` for real). 
in the test I just added for the fetch call, I do not have `disableLifecycleMethods` b/c I'm actually testing the fetch.

Reply:
sounds good, thanks for explaining!